cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -218,6 +218,8 @@ The *Update Available* column in the *Private Space* page shows the status of in
 |Upgrading |The private space is undergoing an infrastructure upgrade, either scheduled, self-triggered, or as part of a monthly infrastructure upgrade. 
 |Updating |The private space is undergoing an update that was triggered by a configuration change. 
 |Failed |The private space experienced a failure which can be related to a user configuration update, or to an infrastructure upgrade, either scheduled, self-triggered, or as part of a monthly infrastructure upgrade. 
+|Degraded |The private space is in degrade state.
```

Comment:
```suggestion
|Degraded |The private space is in degraded state.
```
I'm not sure if the two usages of "degraded" should match? Please disregard if not.